commercesearch/infrastructure/productsearch/inmemoryrepository.go

```diff
@@ -20,19 +22,19 @@ type (
 
 	// InMemoryProductRepository serves as a Repository of Products held in memory
 	InMemoryProductRepository struct {
-		//marketplaceCodeIndex - index to get products from marketplaceCode
+		// marketplaceCodeIndex index to get products from marketplaceCode
 		marketplaceCodeIndex map[string]productDomain.BasicProduct
 
-		//attributeReverseIndex - index to get products from attribute
+		// attributeReverseIndex index to get products from attribute
 		attributeReverseIndex map[string]map[string][]string
```

Comment:
this index is a mapping from attribute to marketplace code, right? I'm not sure if it's helpful to add this to the naming or alternatively to add this to the comments that it maps to the product via the marketplace code

Reply:
updated comment